src/facade/dragonfly_connection.h

```diff
@@ -328,8 +328,11 @@ class Connection : public util::Connection {
     util::fb2::EventCount ec;
     std::atomic_size_t subscriber_bytes = 0;
 
+    util::fb2::CondVarAny pipeline_cnd;
```

Comment:
Now I'd add a comment what ec and pipeline_cnt are, because the comment above if QueueBackpressure is outdated now, it serves one more role